lib/services/faceRecognition.js

```
@@ -0,0 +1,131 @@
+var tf = require('@tensorflow/tfjs');
+require('@tensorflow/tfjs-react-native');
+var FaceRecognitionService = /*#__PURE__*/function () {
+  function FaceRecognitionService() {
+    _classCallCheck(this, FaceRecognitionService);
+  }
+  return _createClass(FaceRecognitionService, [{
+    key: "loadModels",
+    value: function () {
+      var _loadModels = _asyncToGenerator(/*#__PURE__*/_regeneratorRuntime().mark(function _callee(modelConfig) {
+        return _regeneratorRuntime().wrap(function _callee$(_context) {
+          while (1) switch (_context.prev = _context.next) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)